python/lsst/pipe/tasks/calibrate.py

```diff
@@ -585,8 +585,9 @@ def copyIcSourceFields(self, icSourceCat, sourceCat):
             self.log.warn("copyIcSourceFields doing nothing because icSourceFieldsToCopy is empty")
             return
 
-        closest = False  # return all matched objects
-        matches = afwTable.matchXy(icSourceCat, sourceCat, self.config.matchRadiusPix, closest)
+        mc = afwTable.MatchControl()
+        mc.findOnlyClosest = False # return all matched objects
```

Comment:
Please run flake8 linter on the changed code. In this case I expect it to complain about needing two spaces before the `#`.